src/runtime/relax_vm/hexagon/builtin.cc

```diff
@@ -53,7 +52,8 @@ TVM_REGISTER_GLOBAL("vm.builtin.hexagon.dma_copy")
     });
 
 TVM_REGISTER_GLOBAL("vm.builtin.hexagon.dma_wait")
-    .set_body_typed([](TVMArgValue vm_ptr, int queue_id, int inflight_dma) {
+    .set_body_typed([](TVMArgValue vm_ptr, int queue_id, int inflight_dma, NDArray src_arr,
+                       NDArray dst_arr) {
```

Comment:
For `src_arr` and `dst_arr`, perhaps we should add the [`[[maybe_unused]]`](https://en.cppreference.com/w/cpp/language/attributes/maybe_unused).

Reply:
Thank you @quic-sanirudh ! I have fixed this in the latest patch.